app/controllers/users_controller.rb

```diff
@@ -9,4 +10,9 @@ def show
     @user = User.find(params[:id])
   end
 
+  private
+
+  def call_logger2
+    logger.debug 'UsersController log'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.